source/guides/getting-started.md

```diff
@@ -159,6 +159,16 @@ Note that, although Hanami is ready for a Behavior Driven Development workflow o
 We'll go with [Minitest](https://github.com/seattlerb/minitest) here (which is the default), but we can use [RSpec](http://rspec.info) by creating the project with `--test=rspec` option.
 Hanami will then generate helpers and stub files for it.
 
+Nonetheless we'll have to setup the test database to be able to run the tests.
+First we have to define our database url in `.env.test`.
+Now our database is configured we have to create it and migrate our schema by running
+
+```shell
+% HANAMI_ENV=test bundle exec db prepare
```

Comment:
@Bounga `db prepare` is `db drop` + `db create` + `db migrate`. Because we already have created the database this should be `migrate` instead of `prepare` 😉